main.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'app'
+
+def main
+  app = App.new
+
+  puts '📖 Welcome to School Library 📖'
+  puts ''
+  # app.present_options
+  # app.check_input
```

Comment:
- [ ] Kindly remove commented code. We can use commit history to find old code if we want.